### ISSUE TITLE
event StoreHeader can have a returnCode collision

### ISSUE THREAD

**Issue by ethers** [ethereum/btcrelay#34]
StoreHeader for the header for blockheight say 10010 will lead to ambiguty with ERR_DIFFICULTY https://github.com/ethereum/btcrelay/blob/develop/constants.se#L15

maybe error codes for storeBlockHeader should all be negative numbers.

**Comment by staccDOTsol** [comment]
Yes, you are correct. The event `StoreHeader` in Ethereum can potentially have a collision with the `ERR_DIFFICULTY` error code. This is because both the event and the error code are defined with the same name in the `constants.se` file of the `btcrelay` repository.

To avoid this ambiguity, one possible solution could be to use negative numbers for the error codes related to the `storeBlockHeader` function. By using negative numbers, you can differentiate them from the positive return codes of the `StoreHeader` event.

It's important to ensure that the error codes are clearly defined and documented to avoid any confusion or conflicts in the codebase.
